documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -13,11 +13,17 @@ When applied, the configuration creates a dedicated ingress and service for an e
 Clients connect to the bootstrap ingress, which routes them through the bootstrap service to connect to a broker. 
 Per-broker connections are then established using DNS names, which route traffic from the client to the broker through the broker-specific ingresses and services.
 
-To connect to a broker, you specify a hostname for the ingress bootstrap address, as well as the TLS certificate.
-Authentication is optional.
-
+To connect to a broker, you specify a hostname for the ingress bootstrap address, as well as the TLS certificate used for authentication.
```

Comment:
are we sure about "TLS certificate used for authentication" ? I mean if we mean "server authentication" we should clarify it, otherwise "...for encryption" (which to me implies server authentication).
I don't think you were referring to mTLS (so adding client authentication), or? Even because later on we talk about enabling client authentication if you want.

Reply:
Thanks @ppatierno 
I think we should say here what you might need to configure in your client related to the server. So maybe we should change to: _you specify a hostname for the ingress bootstrap address, as well as the certificate used to verify the authenticity of the broker._

Reply:
I would say " ... as well as the certificate for TLS encryption". When you talk about TLS encryption, it's implicit you are authenticating the server. A TLS channel is encrypted with server authentication as well. Only when you add the client authentication we can talk about mTLS.